### PR TITLE
use-new-brancher-app-endpoint

### PR DESCRIPTION
**Pull request by markdentoom**
Change the create and list brancher endpoints to use the new url

### CODE REVIEW

File: tests/unit/Service/BrancherAppTest.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function testCreateBrancherAppWithData()
```
Comment (tdgroot) [inline tests/unit/Service/BrancherAppTest.php:46]: I don't think the mocked responses are correct anymore, or did it stay the same?

File: tests/unit/Service/BrancherAppTest.php [hunk 2]
```diff
@@ -25,7 +25,7 @@ public function testCreateBrancherApp()
```
Comment (tdgroot) [inline tests/unit/Service/BrancherAppTest.php:25]: Here's a mocked response as well

Comment (markdentoom) [reply tests/unit/Service/BrancherAppTest.php:25]: The content of the endpoint should still be the same, it's just the url that changed

File: tests/unit/Service/BrancherAppTest.php (hunk 1 above)
Comment (markdentoom) [reply tests/unit/Service/BrancherAppTest.php:46]: The content of the endpoint should still be the same, it's just the url that changed

**Review by pieterjohannes — APPROVED** [review]

**Pull request merged**